examples/golang/topic-creation/go.mod

```diff
@@ -0,0 +1,7 @@
+module examples/topic-creation
+
+go 1.18
```

Comment:
please use the same `go.mod` as in the "ubject-upload" example.
no need for go v1.18